tests/test_single_topology.py

```diff
@@ -854,7 +854,7 @@ def get_vacuum_solvent_u_grads(ff, lamb):
             vacuum_u_inter_scaled,
             solvent_grad_inter_scaled,
             solvent_u_inter_scaled,
-        ) = get_vacuum_solvent_u_grads(ffs.solv, lamb)
+        ) = get_vacuum_solvent_u_grads(ffs.prot, lamb)
```

Comment:
correct usage, but could look like a bug when reading over it without fresh context. could consider renaming `ffs.prot` to `ffs.env` or similar

Reply:
+1